### PR TITLE
refactor flexural_design

### PR DESCRIPTION
**Pull request by rodrigorahal**

### CODE REVIEW

File: structuraldesign/design/beam_design.py [hunk 1]
```diff
@@ -11,124 +11,132 @@ def __init__(self, beam):
...
-                x = _neutral_line(Mcd_w, b)
+        As_comp = Rsd_neg / self.reinforcement.fyd * 10
 
-                Rcd_w = 0.85 * self.beam.concrete.fcd * 1000 * b * 0.8 * x
+        return As_trac, As_comp, x, xd
 
-            Rcd = Rcd_f + Rcd_w
+    def _minimun_reinforcement(self, section, concrete, ro_min=0.15/100):
+        Md_min = 0.8 * concrete.fctk_sup * section.Wxinf * 1000
+        As_min, _, _, _ = self._necessary_reinforcement(Md_min, 0, None)
```
Comment (StollToaldo) [inline structuraldesign/design/beam_design.py]: Maybe we could get the neutral_line from `self.necessary_reinforcement(Md_min, 0, None)` and return it when we call `_minimum_reinforcement`

File: structuraldesign/detailing/beam_detailing.py [hunk 2]
```diff
@@ -260,7 +260,7 @@ def create_zone_rebars(self, zone, nrebars, fi, al, lb):
 
         zone.set_rebars(rebars)
 
-def place_rebars(rebars, section_stirrup, mpositive=True, sx_vib=0.05, yl=None, n_placed=0):
+def place_rebars(rebars, section_stirrup, zone=True, sx_vib=0.05, yl=None, n_placed=0):
```
Comment (StollToaldo) [inline structuraldesign/detailing/beam_detailing.py]: I think that we will get confused with this "zone" and the "diagram zone". Maybe we should use another name for that.

File: structuraldesign/tests/beam_detailing_test.py [hunk 3]
```diff
@@ -10,7 +10,7 @@
 
 class DiagramTest(unittest.TestCase):
     def setUp(self):
-        self.filename = "structuraldesign/tests/ftool_moment_diagram.txt"
```
Comment (StollToaldo) [inline structuraldesign/tests/beam_detailing_test.py]: This doesn't work in my fucking motherfucker Windows =( I need to solve this shit.

File: structuraldesign/design/beam_design.py (hunk 1 above)
Comment (rodrigorahal) [inline structuraldesign/design/beam_design.py]: Yes, but what if return `ro_min` as the `As_min`?

**Pull request merged**